### DISCUSSION TITLE
webview2 runtime

### DISCUSSION THREAD

**Discussion by AGS5027** [MicrosoftEdge/WebView2Feedback#4025] [Q&A]
Hi,

  I have developed a wpf app by adding webview2 control. I have set up the app by using evergreen distribution for webview2 runtime as prerequisites.
but while running the app , it doesn't show the webview2 control content at all

please help on this

**Comment by novac42** [comment]
@AGS5027 May you provide sample code for us to replicate the issue and provide information of these?
- Framework (WinUI, Win32, WinForms, UWP, WPF, etc)
- SDK and Runtime version.
- Operating System
This will help with figuring out how to best address your issue. Thanks!

**Comment by AGS5027** [comment]
Framework .net 4.7.2
operating system windows 10
sdk version 1.0.1587.40
runtime version downloaded from here https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section (Evergreen)

only using webview2  control source tag not much code in wpf app

adding set up project to build msi installer

Please provide me the steps to use the webview2 control in wpf app and run as desktop icon for production

**Comment by AGS5027** [comment]
Hi any one can help on me on this.
I was unable to do with Evergreen Bootstrapper runtime installer. It would be great if i can get help on this

**Comment by Shangminx** [comment]
@AGS5027 Hi, may you provide some related code? to repro this issue.
You can refer to the wpf sample app for more information. https://learn.microsoft.com/en-us/microsoft-edge/webview2/samples/webview2wpfbrowser
